frontend/src/utils/geojson.ts

```diff
@@ -21,34 +22,27 @@ function toGeoJSON<T>(f: (point: T) => GeoJSON.Feature<GeoJSON.GeometryObject>)
 function providerToFeature(
   point: GeocodedProvider
 ): GeoJSON.Feature<GeoJSON.Point> {
+  // Delete empty properties.
+  Object.keys(point).forEach((key) => (!point[key]) && delete point[key])
```

Comment:
Does it make sense to delete these properties or to hide them? I imagine there could be cases where we'd want to keep a property that's empty (or false-y) on some points.